verticapy/core/vdataframe/_aggregate.py

```diff
@@ -100,16 +100,15 @@ def aggregate(
             |    **mode**: most occurent element
             |    **percent**: percent of non-missing elements
             |    **q%**: q quantile (ex: 50% for the median)
-                                    Use the 'approx_q%' (approximate quantile)
-                                    aggregation to get better performance.
+                         Use the 'approx_q%' (approximate quantile)
+                         aggregation to get better performance.
             |    **prod**: product
             |    **range**: difference between the max and the min
             |    **sem**: standard error of the mean
             |    **skewness**: skewness
             |    **sum**: sum
             |    **std**: standard deviation
-            |    **topk**: kth most occurent element
-                                    (ex: top1 for the mode)
+            |    **topk**: kth most occurent element (ex: top1 for the mode)
             |    **topk_percent**: kth most occurent element density
             |    **unique**: cardinality (count distinct)
             |    **var**: variance
```

Comment:
for the line below, plz update the spacing. the word version should not come under the line of "|". Instead "v" of version should align with "O" of others on the top line.

Reply:
Done but please double check again.